packages/now-cli/src/util/alias/get-inferred-targets.ts

```diff
@@ -7,7 +7,11 @@ export default async function getInferredTargets(
   output: Output,
   config: Config
 ) {
-  output.warn(`The ${cmd('now alias')} command (no arguments) was deprecated in favour of ${cmd('now --prod')}.`);
+  output.warn(
+    `The ${cmd(
+      'now alias'
+    )} command (no arguments) was deprecated in favor of ${cmd('now --prod')}.`
```

Comment:
Switched spelling of "favour" to "favor" (consistent with the warning on line 18).